### PR TITLE
72755 (11.2-U3): Define Docker Host replacement

### PR DESCRIPTION
**Pull request by Mrt134**
Reopen PR so it can be merged when 11.2-U3 work is allowed in the branch.
Replacement defined in conf.py.
Rework virtualmachines.rst to use the replacement.
Fix some rough spots caused by the replacement.
Fix a couple visual errors.
HTML build testing: no issues.
Cherry-pick Warren's additional commits.

### CODE REVIEW

File: userguide/virtualmachines.rst [hunk 1]
```diff
@@ -514,47 +514,47 @@ see the same environment.
...
+.. _Creating |dockerhost|:
 
-Creating Docker Host
-~~~~~~~~~~~~~~~~~~~~
+Creating |dockerhost|
+~~~~~~~~~~~~~~~~~~~~~
 
 :numref:`Figure %s <vms_add_docker_fig>` shows the Wizard that appears
 after going to
 :menuselection:`Virtual Machines`,
```
Comment (a-stjohn) [inline userguide/virtualmachines.rst]:
```suggestion
|ui-add| :menuselection:`--> Virtual Machines`
```

File: userguide/virtualmachines.rst [hunk 2]
```diff
@@ -514,47 +514,47 @@ see the same environment.
...
-clicking |ui-add|, and selecting :guilabel:`Docker Host` as the
+clicking |ui-add|, and selecting |dockerhost| as the
```
Comment (a-stjohn) [inline userguide/virtualmachines.rst]:
```suggestion
and selecting |dockerhost| as the
```

File: userguide/virtualmachines.rst [hunk 3]
```diff
@@ -564,31 +564,31 @@ Docker Host configuration options are described in
...
-   | 2        | Start on Boot      | checkbox       | Set to start this Docker Host when the %brand% system boots.                       |
+   | 2        | Start on Boot      | checkbox       | Set to start this |dockerhost| when the %brand% system boots.                      |
    |          |                    |                |                                                                                    |
    +----------+--------------------+----------------+------------------------------------------------------------------------------------+
-   | 3        | Virtual CPUs       | integer        | Enter the number of virtual CPUs to allocate to the Docker Host. The maximum is 16 |
-   |          |                    |                | unless the host CPU also limits the maximum.                                       |
+   | 3        | Virtual CPUs       | integer        | Enter the number of virtual CPUs to allocate to the |dockerhost|. The maximum is   |
+   |          |                    |                | *16* unless the host CPU also limits the maximum.                                  |
```
Comment (a-stjohn) [inline userguide/virtualmachines.rst]:
```suggestion
   |          |                    |                | *16* unless the host CPU limits the maximum.                                       |
```

File: userguide/virtualmachines.rst [hunk 4]
```diff
@@ -644,38 +644,36 @@ page.
...
+Click |ui-options| and :guilabel:`Serial` to
+log in to the |dockerhost|. Enter :literal:`rancher` for the user name
 and :literal:`docker` for the password.
 
 The default password is changed in the :guilabel:`Devices` by stopping
-the Docker Host, clicking |ui-options|, and :guilabel:`Devices`. Click
+the |dockerhost|, clicking |ui-options|, and :guilabel:`Devices`. Click
 |ui-options| and :guilabel:`Edit` for the :guilabel:`RAW` device and
 enter a new value in the :guilabel:`password` field. Passwords cannot
 contain a space.
```
Comment (a-stjohn) [inline userguide/virtualmachines.rst]:
```suggestion
contain spaces.
```

File: userguide/virtualmachines.rst [hunk 5]
```diff
@@ -644,38 +644,36 @@ page.
...
+   Changing the |dockerhost| Password
 
 
-Start the Docker Host
-~~~~~~~~~~~~~~~~~~~~~
+Start the |dockerhost|
+~~~~~~~~~~~~~~~~~~~~~~
 
 Go to
 :menuselection:`Virtual Machines`
```
Comment (a-stjohn) [inline userguide/virtualmachines.rst]:
```suggestion
:menuselection:`Virtual Machines -->` |ui-options|
```

File: userguide/virtualmachines.rst [hunk 6]
```diff
@@ -644,38 +644,36 @@ page.
...
-and click
-|ui-options| :menuselection:`--> Start`
-to boot the Docker Host.
+and click |ui-options|, then :guilabel:`Start` to boot the |dockerhost|.
```
Comment (a-stjohn) [inline userguide/virtualmachines.rst]:
```suggestion
and click :guilabel:`Start` to boot the |dockerhost|.
```

File: userguide/virtualmachines.rst (hunk 1 above)
Comment (Mrt134) [reply userguide/virtualmachines.rst]: I had to break the navigation steps apart because I'm pointing to a screen within the wizard.

**Comment by Mrt134** [comment]
Updated from the 710 review too.

**Pull request merged**